aws-storage-s3/src/test/java/com/amplifyframework/storage/s3/operation/AWSS3StorageListOperationTest.kt

```diff
@@ -134,6 +135,61 @@ public class AWSS3StorageListOperationTest {
             {}
         )
         awsS3StorageListOperation.start()
-        Mockito.verify(storageService).listFiles(expectedKey, "publicCustom/")
+        Mockito.verify(storageService).listFiles(expectedKey, "publicCustom/", null)
```

Comment:
What the.... this test file is mixing `mockk` and `Mockito`. There should not be any `Mockito` usage in a Kotlin test - it's only to be used for legacy Java tests.

Don't need to fix in this PR but I'm making a note to come back to remove the Mockito usage.

Reply:
I'll make a note to clean this up outside of this PR as well.